core/net/examples/test_net_mk1.rs

```diff
@@ -91,12 +91,32 @@ async fn main() -> Result<()> {
         }
         Side::Sender => {
             let listener_id = Options::id(&Side::Listener);
+            let caller_id: NodeId = "0xbeef000000000000000000000000000000000000"
+                .parse()
+                .unwrap();
             let r = listener_id
                 .try_service(net::PUBLIC_PREFIX)?
-                .send(Test("Test msg".into()))
+                .send_as(caller_id, Test("Test 1 msg".into()))
                 .map_err(Error::msg)
                 .await?;
-            log::info!("Sending Result: {:?}", r);
+            assert!(r.is_ok());
```

Comment:
Maybe we could unit test the root cause of the failure to keep the example cleaner?

Reply:
what do you mean?

Reply:
That we most probably could unit test why an unknown sender (`ya_net::from`) would fail the call, so we could  clean up the assertions. On a second look, the example is brief enough so please disregard this comment

Reply:
this is not unit test, but rather an integration test
I'm adding more comments in form of assert custom messages